src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlOutputFormatter.cs

```diff
@@ -73,5 +76,32 @@ public static XmlWriterSettings GetDefaultXmlWriterSettings()
         {
             return XmlWriter.Create(writeStream, xmlWriterSettings);
         }
+
+        private Type TryGetSerializableErrorType(Type type)
+        {
+            if (type == null)
+            {
+                return null;
+            }
+
+            if (type.Equals(SerializableErrorType))
+            {
+                return SerializableErrorWrapperType;
+            }
+
+            return type;
+        }
+
+        internal object TryWrapSerializableErrorObject(object obj)
```

Comment:
Don't have internal methods. In general looks like there is an issue with the DCS deriving from the XML formatter, we should probably break that, it's definitely wrong. They should not share identity